packages/peregrine/src/Toasts/useToasts.js

```diff
@@ -169,9 +197,11 @@ export const useToasts = () => {
         () => ({
             addToast,
             dispatch,
-            removeToast
+            removeToast,
+            pauseAllToasts,
```

Comment:
If you are going to add functions to the toast api you need to add to the jsdoc above.